docs/migration_guides/v0.10.2_to_v1.0.0_nearcore.md

```diff
@@ -14,7 +14,7 @@ index f38e88411..1587f4131 100644
 --- a/Cargo.toml
 +++ b/Cargo.toml
 -borsh = { version = "0.10.2", features = ["rc"] }
-+borsh = { version = "1.0.0-alpha.5", features = ["derive", "rc"] }
++borsh = { version = "=1.0.0-alpha.5", features = ["derive", "rc"] }
```

Comment:
otherwise, the version is interpreted as `^1.0.0-alpha.5` and `1.0.0-alpha.6` is picked up, which has `try_to_vec` already removed, not merely deprecated